### ISSUE TITLE
Camera frustum plotting

### ISSUE THREAD

**Issue by mklingen** [personalrobotics/or_rviz#7]
When adding a sensor to an OpenRAVE robot, the frustums get plotted wrongly:

![Imgur](http://i.imgur.com/J9PorSN.png)

This seems to be a problem with the line plotting functionality in `or_interactivemarker`

**Comment by mkoval** [comment]
Can you post a script (inline or a Gist would be fine) that reproduces this?

My hunch is that this is a mismatch in units on line width. I believe OpenRAVE expects lines to be specified in pixels and RViz expects lines to be specified in meters.

**Comment by mklingen** [comment]
Yes, I think that is the issue. This was the openravepy example for visibility planning with viewer set to or_rviz

**Comment by mklingen** [comment]
Will be fixed in or_interactivemarker now.
